src/test/test_strategy.py

```diff
@@ -9,8 +10,8 @@
 
 class TestPortfolio(unittest.TestCase):
     RESOLUTION = Resolution.DAILY
-    START_TIME = datetime(2019, 3, 5)
-    END_TIME = datetime(2019, 4, 22)
+    START_TIME = datetime(2020, 7, 6)
```

Comment:
I would prefer if existing tests are not modified, unless indicators break existing functionality. It would probably be better to create a new test file for indicators.

Reply:
Also, don't change the dates because it will break all the current tests